### PR TITLE
Feature/add details popup

### PR DESCRIPTION
**Pull request by VargaElod23**

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,36 +1,34 @@
-import React from "react";
-import "./App.css";
-import Header from "./components/header/Header";
-import { useMediaQuery } from "react-responsive";
-import useMetamask from "./hooks/useMetamask";
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
-import { MetaMaskProvider } from "metamask-react";
-import Home from "./pages/Home/Home";
+import './App.css';
```
Comment (Malak67) [inline client/src/App.tsx:1]: Don't forget to pull latest changes in Develop and fix any possible conflicts.

File: client/src/components/modals/ExtendedCardDetails.modal.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  justify-content: space-between;
+  align-items: baseline;
+  margin-top: 0;
+`;
+
+const ButtonDiv = styled.div`
+  cursor: pointer;
+`;
+
+const CardDetailsModal = (props: ExtendedDetailsProps) => {
```
Comment (Malak67) [inline client/src/components/modals/ExtendedCardDetails.modal.tsx:72]: I would extract the props directly here than in a new line below
`const CarddetailsModal = ({cardData} : ExtendedDetailsProps)`

File: client/src/pages/Home/Home.tsx [hunk 3]
```diff
@@ -109,89 +112,90 @@ const TitleText = styled.h3`
...
+  const { setOpen, close, isOpen } = useModal();
   useEffect(() => {
     setCardData([
       {
         title: 'ApeCoin Staking Launch!',
         description:
           'APE is launching it’s testnet, and we’d like everyone to come & check it out! All participants will be able to claim rewards as loooooooong All participants will be able to claim rewards as loooooooongAll participants will be able to claim rewards as loooooooongAll participants will be able to claim rewards as loooooooongAll participants will be able to claim rewards as loooooooongAll participants will be able to claim rewards as loooooooongAll participants will be able to claim rewards as loooooooongAll participants will be able to claim rewards as loooooooongAll participants will be able to claim rewards as loooooooongAll participants will be able to claim rewards as loooooooongvvvAll participants will be able to claim rewards as loooooooongAll participants will be able to claim rewards as loooooooongAll participants will be able to claim rewards as loooooooongAll participants will be able to claim rewards as loooooooongv',
-        pool: 100000,
+        poolAmount: 100000,
```
Comment (Malak67) [inline client/src/pages/Home/Home.tsx:140]: This needs to match the backend entity. So if changed here, we need to change it there as well (new migration etc.). if not that important, I say we keep it as it was so we don't do the API changes but it is important we can change it. Just need to remember that.

File: client/src/pages/Home/Home.tsx [hunk 4]
```diff
@@ -109,89 +112,90 @@ const TitleText = styled.h3`
...
         poolToken: 'APE',
-        bonus: 12000,
+        bonusAmount: 12000,
+        creatorAddress: '0x000000000000000000000000000000',
```
Comment (Malak67) [inline client/src/pages/Home/Home.tsx:143]: The entity has this property named as accountAddress. Same comment as above.

File: client/src/pages/Home/Home.tsx [hunk 5]
```diff
@@ -109,89 +112,90 @@ const TitleText = styled.h3`
   color: #000000;
 `;
 
-interface CardData {
+interface CardDataDummy {
```
Comment (Malak67) [inline client/src/pages/Home/Home.tsx:115]: Not sure this name is ok. Will we change it later?

File: client/src/pages/Home/Home.tsx [hunk 6]
```diff
@@ -1,13 +1,16 @@
...
-import Card from '../../components/card/Card';
+import Card, { CardData } from '../../components/card/Card';
 import Input from '../../components/input/Input';
 import InfiniteScroll from 'react-infinite-scroll-component';
 import './Home.css';
 import LoadingSpinner from '../../assets/icons/Spinner';
 import styled from 'styled-components';
 import NotFoundIcon from 'src/assets/icons/NotFount';
+import { useModal } from 'src/hooks/useModal';
+import CardDetailsModal from 'src/components/modals/ExtendedCardDetails.modal';
```
Comment (Malak67) [inline client/src/pages/Home/Home.tsx:12]: I think we should have a consistency in the file name and component name.

File: client/src/pages/Home/Home.tsx (hunk 4 above)
Comment (VargaElod23) [reply client/src/pages/Home/Home.tsx:143]: Will be refactored anyways

**Pull request merged**